src/components/ButtonToolbar/ButtonToolbar.scss

```diff
@@ -1,9 +1,3 @@
-/* ==========================================================================
```

Comment:
zouden we dit niet laten staan? Het help de component wat uit te leggen. Mss moeten we het wel verplaatsen naar de tsx file van de component